internal/characters/baizhu/skill.go

```diff
@@ -77,13 +78,15 @@ func (c *char) chain(src int, count int) combat.AttackCBFunc {
 	}
 	return func(a combat.AttackCB) {
 		//on hit figure out the next target
-		next := c.Core.Combat.RandomEnemyWithinArea(combat.NewCircleHitOnTarget(a.Target, nil, 10), func(t combat.Enemy) bool {
-			return a.Target.Key() != t.Key()
-		})
+		next := c.Core.Combat.RandomEnemyWithinArea(combat.NewCircleHitOnTarget(a.Target, nil, 10), nil)
 		if next == nil {
 			c.skillHealing()
 			return
 		}
+		delay := skillTickInterval
+		if next.Key() == a.Target.Key() {
+			delay += 6 // add some delay in case it's a different target
+		}
```

Comment:
- The comment here doesn't seem to match what's happening with the `if` check. It looks like we're adding delay if the target stays the same.
- Should we document this 6f delay in the baizhu frame sheet?

Reply:
I wouldn't. All the values here are dependent on distance so I just picked some reasonable value from a quick test in-game